plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -826,7 +826,6 @@ fu_dell_dock_ec_write_fw(FuDevice *device,
 
 	/* meet the minimum EC version */
 	if ((flags & FWUPD_INSTALL_FLAG_FORCE) == 0 &&
-	    (self->data->module_type != MODULE_TYPE_130_USB4) &&
```

Comment:
This is intentional, right?

Reply:
Yes, it was a quirk for the new dock during development phase.